filters/stats/StatsFilter.cpp

```diff
@@ -182,6 +180,39 @@ void StatsFilter::extractMetadata()
         t.add("position", position++);
         s.extractMetadata(t);
     }
+
+    // If we have X, Y, & Z dims, output bboxes
+    auto xs = m_stats.find(Dimension::Id::X);
```

Comment:
I'm 50/50 on requiring Z's. My first pass didn't require it, and my second pass did. Interested in others' thoughts on the topic...